app/src/main/java/com/kickstarter/ui/activities/MessagesActivity.java

```diff
@@ -236,6 +230,20 @@ private void requestFocusAndOpenKeyboard() {
     this.getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_ALWAYS_VISIBLE);
   }
 
+  /**
+   * Sets an OffsetChangedListener for the view's AppBarLayout to:
+   * 1. determine the toolbar's alpha based on scroll range
+   * 2. adjust the view's bottom padding via inputs
+   */
+  private void setAppBarOffsetChangedListener(final @NonNull AppBarLayout appBarLayout) {
+    appBarLayout.addOnOffsetChangedListener((layout, offset) -> {
+      this.projectNameToolbarTextView.setAlpha(Math.abs(offset) / layout.getTotalScrollRange());
```

Comment:
FYI this line is just the same as [what we use in our ToolbarUtils](https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/libs/utils/ToolbarUtils.kt#L15)